tests/e2e/gpu-feature-discovery_test.go

```diff
@@ -143,9 +143,9 @@ var _ = NVDescribe("GPU Feature Discovery", func() {
 			// Delete Helm release
 			err := helmClient.UninstallReleaseByName(helmReleaseName)
 			Expect(err).NotTo(HaveOccurred())
-			// cleanup node
+			// Cleanup node
 			common.CleanupNode(ctx, f.ClientSet)
-			common.CleanupNodeFeatureRules(ctx, nfdClient, f.Namespace.Name)
+			common.CleanupNodeFeatures(ctx, nfdClient, f.Namespace.Name)
```

Comment:
Question: Since we delete both `NodeFeatures` and `NodeFeatureRules` in this function is there a better name for the collection of these two?

Reply:
`NFDObjects` ?